scripts/mkcloud

```diff
@@ -56,6 +56,17 @@ mkcconf=mkcloud.config
 rm -f $mkcconf # we dont want to source old info in the line below
 source $qa_crowbarsetup
 
+: ${admin_image_password:='linux'}
+: ${architectures:='aarch64 x86_64 s390x'}
+: ${clouddata_nfs:=$clouddata}
+: ${clouddata_nfs_dir:='srv/nfs'}
+: ${repopath_sles12sp1:='repos/$arch/SLES12-SP1-Pool'}
+: ${repopath_sles12sp1up:='repos/$arch/SLES12-SP1-Updates'}
+: ${manila_service_image_path:='images/other'}
```

Comment:
Less variables is a good idea 😄 
Looking forwarding to give the refactored version a try. Important for us is just to have a way to define our folder structure and paths for repo and nfs server.